File: python/7 kyu/337.py
# https://www.codewars.com/kata/5d1e1560c193ae0015b601a2/train/python

# Passed

def count(a, t, x):
    absolute_differences = [abs(num - t) for num in a]
    result = len([1 for diff in absolute_differences if (x != 0 and diff % x == 0 ) or (x == 0 and diff == 0)])
    return result

output = count([11, 5, 3], 7, 2)
print(output)

File: nodejs/7 kyu/337.js
// https://www.codewars.com/kata/5d50e3914861a500121e1958/train/javascript

// Passed

function addLetters(...letters) {
  let allLetters = "abcdefghijklmnopqrstuvwxyz";
  let letterToValueMap = new Object();
  let valueToLetterMap = new Object();
  let value = 0;
  for (const letter of allLetters) {
    value++;
    letterToValueMap[letter] = value;
    valueToLetterMap[value] = letter;
  }

  let key =
    letters
      .map((letter) => letterToValueMap[letter])
      .reduce((prev, curr) => prev + curr, 0) % 26;
  key = key === 0 ? 26 : key;
  let result = valueToLetterMap[key];
  return result;
}

const output = addLetters("a", "b", "c");
console.log(output);